delfin/alert_manager/trap_receiver.py

```diff
@@ -32,10 +32,8 @@
 
 LOG = log.getLogger(__name__)
 
-# Currently static mib file list is loaded
-# Mechanism to be changed to load all mib file
-MIB_LOAD_LIST = ['SNMPv2-MIB', 'IF_MIB', 'EMCGATEWAY-MIB', 'FCMGMT-MIB',
-                 'ISM-HUAWEI-MIB']
+# Mib file list, can be .mib or .py(precompiled) format
+MIB_LOAD_LIST = ['EMCGATEWAY-MIB', 'FCMGMT-MIB', 'ISM-HUAWEI-MIB']
```

Comment:
Why need MIB_LOAD_LIST here?

Reply:
It is possible that we can place all files in the path and code can scan all files and load internally.
But to avoid problem with dumping any unnecessary files in the folder, we are controlling loading of necessary mibs, whats the opinion

Reply:
here should be addressed

Reply:
@wisererik , addressed